the import is not supposed to be used,
and from what i can see it works alright without it

https://github.com/pyca/cryptography/issues/10609

diff --git a/spsdk/crypto/__init__.py b/spsdk/crypto/__init__.py
index c587355..7e98b8c 100644
--- a/spsdk/crypto/__init__.py
+++ b/spsdk/crypto/__init__.py
@@ -50,3 +50,2 @@ from cryptography.hazmat.backends import *
 from cryptography.hazmat.backends import default_backend
-from cryptography.hazmat.backends.openssl.rsa import *
 
